src/pc/djui/djui_hud_utils.c

```diff
@@ -386,6 +416,8 @@ void djui_hud_print_text_interpolated(const char* message, f32 prevX, f32 prevY,
     interp->height = font->defaultFontScale;
     interp->z = savedZ;
     interp->resolution = sResolution;
+    interp->rotation = sRotation;
+    
```

Comment:
pointless new line?

Reply:
I agree

Reply:
assuming that first line wasn't pointless since this happens after its removal
![image](https://github.com/user-attachments/assets/4d20729b-4a2a-4eb9-9faf-4e68799dcb9f)